packages/default/scss/appbar/_layout.scss

```diff
@@ -0,0 +1,68 @@
+@include exports("appbar/layout") {
+
+    // Appbar
+    .k-appbar {
+        @include use-background-clip();
+        margin: $appbar-margin-y $appbar-margin-x;
+        padding: $appbar-padding-y $appbar-padding-x;
+        border-width: $appbar-border-width;
+        border-style: solid;
+        box-sizing: border-box;
+        outline: 0;
+        font-family: $appbar-font-family;
+        font-size: $appbar-font-size;
+        line-height: $appbar-line-height;
+        list-style: none;
+        display: flex;
+        flex: 0 0 100%;
+        flex-direction: row;
+        flex-wrap: wrap;
+        align-items: center;
+        overflow: hidden;
+        -webkit-touch-callout: none;
+        -webkit-tap-highlight-color: $rgba-transparent;
+
+        &.k-appbar-sticky {
+            width: 100%;
+            position: sticky;
+            top: 0;
+            z-index: 1;
+        }
+
+        &.k-appbar-top {
+            top: 0;
+            bottom: auto;
+        }
+
+        &.k-appbar-bottom {
+            top: auto;
+            bottom: 0;
+        }
+
+        &.k-appbar-inherit {
```

Comment:
@joneff do we want to support `inherit` background of the appbar?

Reply:
After offline discussion with @joneff, we decided to have themeColor: inherit option that will apply k-appbar-inherit class name, but there should be no styles for it. So I'll remove these lines from the PR.